### ISSUE TITLE
install-sx.sh tries to install obbalancer 

### ISSUE THREAD

**Issue by PabloCastellano** [spesmilo/sx#102]
@genjix removed obblanacer code in March 2014 but install-sx.sh tries to install it.
I missed why it was removed. Isn't it needed anymore and should I clean the script or maybe there's some replacement?

**Comment by PabloCastellano** [comment]
Fixed in this pull request: https://github.com/libbitcoin/obelisk/pull/20

**Comment by veox** [comment]
Yup, there's now only the `obelisk-server` binary (PKA `obworker`).

Not sure as to @genjix's reasons, but IMO connection/traffic balancing is better done on a lower level anyway.

**Comment by genjix** [comment]
hey, obelisk is now obsoleted. the new project is libbitcoin-server. I've updated the install-sx.sh script here.
Also libbitcoin-explorer is the new repo/rewrite for this project made by Eric Voskuil.. he's been doing a lot of work on that.
